webpack.config.babel.js

```diff
@@ -34,7 +35,12 @@ export default {
       {
         test: /\.js$/,
         loader: 'babel-loader',
-        include: [ PATH_SRC, PATH_TESTS ]
+        include: [PATH_SRC, PATH_TESTS, VIDEOJS_VAST]
+      },
+      {
+        test: /\.(eot|woff|woff2|svg|ttf)([\?]?.*)$/,
+        loader: 'file-loader',
+        include: [VIDEOJS_VAST]
```

Comment:
why are we doing this? i don't see any eot/woff/svg/etc files in the maestro-videojs-vast repo